doc/01-installation.md

```diff
@@ -0,0 +1,140 @@
+# Shopware OpenTelemetry configuration
+
+This document explains how to install and configure OpenTelemetry for Shopware to collect and transport traces, metrics, and logs. We will use OTLP over gRPC as the transport protocol.
+
+## Installation
+
+1. Install dependencies
+
+    ```shell
+    composer require open-telemetry/sdk open-telemetry/exporter-otlp open-telemetry/transport-grpc
+    composer require open-telemetry/opentelemetry-logger-monolog # if logs forwarding is needed
+    composer require shopware/opentelemetry
+    ```
+
+   To use the latest version of the shopware/opentelemetry package, you can modify the corresponding line in composer.json to:
+    ```json
+    "shopware/opentelemetry": "dev-main as 1.0.0"
+    ```
+
+2. Create the config file config/packages/opentelemetry.yaml with the following content:
+
+    ```yaml
+    # metrics
+    open_telemetry_shopware:
+        metrics:
+            enabled: true
+            namespace: 'my.prefix.for.metrics'
+
+    # traces
+    shopware:
+        profiler:
+            integrations:
+                - OpenTelemetry
```

Comment:
Might be worth mentioning that this setting is only relevant for using the `Profiler`, not the `auto` tracing in general.

Auto tracing would be working directly after requiring the dependency via composer, without any configuration updates (besides the env vars).